app/config.py

```
@@ -0,0 +1,12 @@
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+class Config:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)